services/movie.py

```diff
@@ -0,0 +1,44 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        return queryset.filter(
+            genres__id__in=genres_ids
+        ).filter(
+            actors__id__in=actors_ids
+        )

```

Comment:
not queryset.filter(
            genres__id__in=genres_ids
        ).filter(
            actors__id__in=actors_ids
        )
        but queryset.filter(
            genres__id__in=genres_ids, actors__id__in=actors_ids
        )